xswap/preprocessing.py

```diff
@@ -0,0 +1,106 @@
+import pathlib as _pathlib
```

Comment:
Why the `_pathlib`? I'm guessing so it doesn't show up in the namespace to external modules? Although I don't usually see this done for module imports